### PR TITLE
Silence is golden

### PR DESCRIPTION
**Pull request by boennemann**
http://www.linfo.org/rule_of_silence.html

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -1,8 +1,11 @@
 module.exports = spawnPouchdbServer
 
+var fs = require('fs')
+
```
Comment (gr2m) [inline index.js:4]: why the empty line? just wondering :)

**Comment by gr2m** [comment]
looks good so far! And thanks for the "rule of silence" link! Will read that :)

File: index.js (hunk 1 above)
Comment (boennemann) [inline index.js:4]: I'm just used to separating core, npm and files.

Comment (jo) [inline index.js:4]: :+1:

Comment (gr2m) [inline index.js:4]: ah good one, cheers!

**Comment by gr2m** [comment]
looks all good to me :+1: merge at will

**Comment by christophwitzko** [comment]
LGTM :+1:

**Pull request merged**